roles/puppet_repositories/molecule/redhat/verify.yml

```diff
@@ -3,7 +3,12 @@
   hosts: all
   gather_facts: false
   tasks:
-    - name: Ensure puppet release RPM is installed
+    - name: Ensure puppet6-release RPM is installed
       package:
         name: puppet6-release
         state: present
+
+    - name: Ensure puppet5-release RPM is absent
```

Comment:
Should this be the first test? You have it the other way round in debian/verify.yml (nitpick, obviously)